terraform/README.md

```diff
@@ -0,0 +1,72 @@
+# Kubernetes Investigations Terraform code
+
+This Readme includes the Terraform code part for the Kubernetes investigations repository.
+The following directory is structured into:
+
+```
+├── README.md
+├── cloud-platform
+├── global-resources
+└── modules
+```
+## Cloud Platform
+The cloud-platform directory is were the cluster specific code lives and relies on Terraform workspaces as the starting point.
+
+## Global Resources
+The global-resources directory contains Terraform code to build the common resources for all clusters.
+Within Global Resources you can find:
+
+### ecr_credentials.tf
+
+This Terraform file uses the following Terraform module which creates ECR credentials and repository on AWS. This Module is used to create the user, ECR repository and credentials for our example app. The team name is `webops` and the repository name is `cloud-platform-reference-app` and these are the parameters used by the module in this case
```

Comment:
Perhaps it's better to just refer to https://github.com/ministryofjustice/cloud-platform-terraform-ecr-credentials/blob/master/README.md ? Correct me if I'm wrong but this section seems to be duplicating the docs from the repo. I think it would be better to just explain how the user should handle code changes here (make a new file / expand `ecr_credentials.tf`) and leave the details in the ecr module repo.

Reply:
this is a specific usage of the module in this case for the demo app, i think that should be documented here.